txnbuild/revoke_sponsorship.go

```diff
@@ -28,30 +28,30 @@ type RevokeSponsorship struct {
 	// Account Id (strkey)
 	Account   *string
 	TrustLine *TrustLineId
-	Offer     *OfferId
-	Data      *DataId
+	Offer     *OfferID
+	Data      *DataID
 	// Claimable Balance Id
 	ClaimableBalance *string
-	Signer           *SignerId
+	Signer           *SignerID
 }
 
 type TrustLineId struct {
```

Comment:
Want to change this one too?
```suggestion
type TrustLineID struct {
```